Tribler/Test/Core/test_permid.py

```diff
@@ -6,9 +6,11 @@
 from M2Crypto.EC import EC
 
 from Tribler.Core import permid
+from Tribler.dispersy.crypto import LibNaCLSK
 from Tribler.Test.Core.base_test import TriblerCoreTest
 
 
+
```

Comment:
iirc, PEP8 specifies that there have to be two lines between the imports and class decoration so I think this line should be removed.